docs/source/app_developers_guide.rst

```diff
@@ -3,27 +3,32 @@
 Application Developer's Guide
 *****************************
 
-This guide covers development of applications which run on top of the
+This guide describes how to develop applications which run on top of the
 Hyperledger Sawtooth :term:`core platform<Sawtooth core>`, primarily through
-the use of Sawtooth's provided SDKs and :term:`REST API`. Topics include
-development of :term:`transaction families<Transaction family>` and associated
-client programs. Transaction families codify business rules used to modify
-state, while client programs typically submit transactions and view state.
+the use of Sawtooth's provided SDKs and :term:`REST API`.
 
-SDKs are provided in several languages: C++, Go, Java, Javascript, and Python.
+Topics include developing a :term:`transaction family<Transaction family>` and
+an associated client program. Transaction families codify business rules used
+to modify state, while client programs typically submit transactions and view
```

Comment:
If we are going to use TF as the umbrella over client, server, and data model then this description should be updated. It appears to be more of a string replace on TP

Reply:
If we are going to use TF as the umbrella over client, server, and data model then this description should be updated. It appears to be more of a string replace on TP